### PR TITLE
Create alertmanager.js

### PR DESCRIPTION
**Pull request by HarHarLinks**
Signed-off-by: Kim Brose <2803622+HarHarLinks@users.noreply.github.com>

### CODE REVIEW

File: contrib/alertmanager.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+function statusBadge(status, severity) {
```
Comment (Half-Shot) [inline contrib/alertmanager.js:1]: I'd:

- Put this in a subdirectory like jsTransformationFunctions
- Add a simple description of what this does in docstring.

**Review by Half-Shot — CHANGES_REQUESTED** [review]
Thanks for this one :). Just a couple of requests from me.

**Comment by HarHarLinks** [comment]
@erebion thanks for the screenshots.
I agree with your sentiment. However I'm not actively using this (just copy-pasted from the hookshot channel), I'm not even running Alertmanager, yet...
I think it'd be best if you could suggest changes via code review?

File: contrib/jsTransformationFunctions/alertmanager.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    if (status === "resolved") {
+        return `<font color='green'><b>[RESOLVED]</b></font>`;
+    }
+
+    switch(severity) {
+        case 'resolved':
+        case 'critical':
+            return `<font color='red'><b>[FIRING - CRITICAL]</b></font>`;
+        case 'warning':
+            return `<font color='orange'><b>[FIRING - WARNING]</b></font>`;
```
[the remark this answers is not in the document]
Comment (Half-Shot) [reply contrib/jsTransformationFunctions/alertmanager.js:28]: I'm not a massive fan of `<br>`. Can we paragraph the event content instead with `<p>` and have the client give it appropriate spacing?

**Comment by HarHarLinks** [comment]
```
"formatted_body": "<p><font color='green'><b>[RESOLVED]</b></font></p><p><b>HostSystemdServiceCrashed</b>: SystemD service crashed<br/>  VALUE = 1<br/>  LABELS: map[__name__:node_systemd_unit_state instance:localhost:9100 job:host_node_exporter name:grafana.service state:failed type:simple]</p><p><a href=\"http://erebion-matrix:9090/graph?g0.expr=node_systemd_unit_state%7Bstate%3D%22failed%22%7D+%3D%3D+1&g0.tab=1\">source</a> | <a href=\"http://localhost:9093#silences/new?filter={alertname%3D%22HostSystemdServiceCrashed%22,instance%3D%22localhost%3A9100%22,job%3D%22host_node_exporter%22,name%3D%22grafana.service%22,severity%3D%22warning%22,state%3D%22failed%22,type%3D%22simple%22}\">silence</a></p>"
```
looks like
image

**Pull request merged**